### PR TITLE
Add to show List of devices in frontend

### PR DESCRIPTION
**Pull request by EdwinKuo1337**
Add rabbitMQ to get device connection list from adaptor
Add rabbitMQ command line function to send dummy commands to adaptor

### CODE REVIEW

**Comment by bingzhang** [comment]
Can you delete the commented out code, please?

And rename the rpc_server.py to adaptor.py and move it to the `script` folder.

**Comment by bingzhang** [comment]
can you also delete __pycache__, maybe add it to gitignore?

File: utilities/user_utilities.py [hunk 1]
```diff
@@ -109,6 +112,18 @@ def get_all_user_events_count(group_ids, select_status, start=None, end=None):
                                         # "createdByGroupId": {"$in": group_ids},
                                         "eventStatus": {"$in": select_status}}))
 
+def get_all_device_status_pagination(skip, limit):
+    statusRpc = RpcClient()
```
Comment (bingzhang) [inline utilities/user_utilities.py:116]: We donot need to create receiver every funtion call.
And can you rename RpcClient?

File: scripts/adaptor.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                     body=status)
+    ch.basic_ack(delivery_tag=method.delivery_tag)
+
+def on_command_request(ch, method, props, body):
+    json_body = json.loads(body)
+    print(f" [.] incomming command: {json_body}")
+    # {'command_id': str(uuid.uuid4()), 'command': 'test_command', 'device_id': 'id_1'}
+    device_id = json_body['device_id']
+    command_id = json_body['command_id']
+    device_status = generate_command_status()
```
Comment (bingzhang) [inline scripts/adaptor.py:57]: Can you put a swtich on the device_commands?

Comment (EdwinKuo1337) [reply scripts/adaptor.py:57]: I'm sorry I don't understand, what do you mean a switch?

Comment (EdwinKuo1337) [reply scripts/adaptor.py]: Added some dummy if else for different dummy commands

**Pull request merged**